website/docs/packages/reference/query-module.md

```diff
@@ -1,6 +1,17 @@
-# App-Level Package Settings
+# Module Instance
 
-This page provides information about the package settings and properties available within the App, which allows you to configure the query and JS modules.
+
+
+When you incorporate a module from the package into your application, you create a module instance. You can create multiple instances of the same package, each with different settings and configurations. Instances are named sequentially (e.g., `productutil1`, `productutil2`), allowing for customized functionalities.
+
+This page provides information about the module settings and properties available within the app, which allows you to configure the query and JS modules.
+
+
+<ZoomImage
+  src="/img/modules-diagram.png" 
+  alt=""
+  caption=""
+/>
 
 
 
```

Comment:
What if we change this heading 2 to "Query module instance" so that it's more clear in the documentation the difference between Query module and Query module instance?